packages/palette/src/elements/DonutChart/DonutChart.tsx

```diff
@@ -110,11 +110,7 @@ export const DonutChart: React.FC<DonutChartProps> = ({
     <svg key={"svg"} width={width + margin} height={width + margin}>
       <g transform={`translate(${centerX}, ${centerY})`}>
         {zeroStateArc}
-        <Spring
-          from={animateProps(hasEnteredViewport)}
-          to={animateProps(!hasEnteredViewport)}
-          delay={500}
-        >
+        <Spring from={{ num: -0.1 }} to={animateProps(!hasEnteredViewport)}>
```

Comment:
Same kind of hack but this one fixes the animation triggering problem.

Reply:
Wow really? Nice hack :)

Reply:
haha 🤷‍♂